### PR TITLE
Added emulator path option for virt-install commandline

### PR DESCRIPTION
**Pull request by sathnaga**
Added emulator path option for virt-install, this
will help in case of custom qemu binary to be used for
running libvirt tests

### CODE REVIEW

**Comment by sathnaga** [comment]
@waynesun09 @will-Do could you please help merge if no comments, Thanks in advance :-)

**Comment by PandaWei** [comment]
ACK

File: virttest/libvirt_vm.py [hunk 1]
```diff
@@ -967,6 +967,10 @@ def add_memballoon(help_text, memballoon_model):
         if params.get("virt_install_debug") == "yes":
             virt_install_cmd += " --debug"
 
+        emulator_path = params.get("emulator_path", None)
+        if emulator_path and has_sub_option('boot', 'emulator'):
+            virt_install_cmd += " --boot emulator=%s" % emulator_path
```
Comment (PandaWei) [inline virttest/libvirt_vm.py]: Looks good, but a piece of suggestion
If user specifies the "emulatorpath" and virt-install doesn't support emulator features, we should give warning message.

Comment (sathnaga) [inline virttest/libvirt_vm.py]: @PandaWei finding the PR got merged, will send this suggestion as a seperate PR.

Comment (sathnaga) [inline virttest/libvirt_vm.py]: @PandaWei @will-Do https://github.com/avocado-framework/avocado-vt/pull/746

**Comment by sathnaga** [comment]
@will-Do Thanks a lot :-)

**Pull request merged**